docs/quickstart.md

```diff
@@ -1,321 +1,24 @@
-There are some great instructions for generating a certificate on the Ubuntu website:
-
-[Ubuntu - Create a Self Signed SSL Certificate](https://help.ubuntu.com/14.04/serverguide/certificates-and-security.html)
-
-The last commands you need to run from that tutorial are in the "Installing the Certificate" section:
-
-~~~~ {.sourceCode .bash}
-sudo cp server.crt /etc/ssl/certs
-sudo cp server.key /etc/ssl/private
-~~~~
-
-Once you have finished the instructions at the link above, and these two files are in your /etc/ssl/certs and /etc/ssl/private, you are ready to move on in this guide.
-
-Setup Nginx:
-------------
-
-Security Monkey uses gunicorn to serve up content on its internal 127.0.0.1 address. For better performance, and to offload the work of serving static files, we wrap gunicorn with nginx. Nginx listens on 0.0.0.0 and proxies some connections to gunicorn for processing and serves up static files quickly.
-
-### securitymonkey.conf
-
-Copy the config file into place:
-
-    sudo cp /usr/local/src/security_monkey/nginx/security_monkey.conf /etc/nginx/sites-available/security_monkey.conf
-    sudo ln -s /etc/nginx/sites-available/security_monkey.conf /etc/nginx/sites-enabled/security_monkey.conf
-    sudo rm /etc/nginx/sites-enabled/default
-    sudo service nginx restart
-
-Logging into the UI
+Installation Instructions:
 -------------------
-
-You should now be able to reach your server
-
-![image](images/resized_login_page-1.png)
-
-Loading Data into Security Monkey
---------------------------------
-To initially get data into Security Monkey, you can run the `monkey find_changes` command. This will go through
-all your configured accounts in Security Monkey, fetch details about the accounts, store them into the database,
-and then audit the items for any issues. 
-
-The `find_changes` command can be further scoped to account and technology with the `-a account` and `-m technology` parameters.
-
-*Note:* This is good for loading some initial data to play around with, however, you will want to configure jobs to automatically
-scan your environment periodically for changes.  Please read the next section for details.
-
-🚨 Important 🚨 - Autostarting and Fetching Data
---------------------------------
-At this point Security Monkey is set to manually run. However, we need to ensure that it is always running and automatically
-fetching data from your environment.
-
-Please review the next section titled [Autostarting Security Monkey](autostarting.md) for details. Please note, this section
-is very important and involved, so please pay close attention to the details.
+1. [Launch a server](installation/01-launch-instance.md)
+2. [Create a database instance](02-create-db.md)
+3. [Install Security Monkey on your server instance](03-install-sm.md)
+4. [Populate your Security Monkey with Accounts](04-accounts.md)
+5. [Create an SSL Certificate](05-ssl.md)
+6. [Setup Nginx](06-nginx.md)
+7. [Login to Security Monkey & load data](07-load-data.md)
```

Comment:
One more request: can you add #8 for the autostarting documentation? I think this could be worded like: "Productionalize Security Monkey (Autostarting)"